src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt

```diff
@@ -60,9 +48,7 @@ class GetCaseByCrnTests : IntegrationTestBase() {
       .jsonPath("$.requirements[0].subCategory")
       .isEqualTo("Regular")
       .jsonPath("$.requirements[0].length")
-      .isEqualTo(100)
-      .jsonPath("$.requirements[0].lengthUnit")
```

Comment:
the only breaking change, have raised https://github.com/ministryofjustice/manage-a-workforce-ui/pull/629 to move to the new field

Reply:
So these have to be released together?

Reply:
I want to see what just releasing this does in dev, not sure how nunjucks will handle nulls